### PR TITLE
Fixed password confirmation bug

### PR DESCRIPTION
**Pull request by debikus**
Password confirmation field now will check matching to password field

image

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,6 +19,7 @@ test-report.xml
 /.idea
 .idea
 .DS_Store
+.env
```
Comment (MickhaelSh) [inline .gitignore:22]: remove this, use .env.local

File: src/utils/validations/login.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ export const lastName = (value) => {
 export const confirmPassword = (password, data) => {
   return emptyField(
     password,
+    !password ? 'common.errorMessages.emptyField' : '',
```
Comment (MickhaelSh) [inline src/utils/validations/login.js:22]: why do you need this check here if empty field func has check for value ?

File: src/constants/translations/en/common.json [hunk 3]
```diff
@@ -69,4 +69,4 @@
     "spanish": "Spanish",
     "arabic": "Arabic"
   }
-}
+}
```
Comment (MickhaelSh) [inline src/constants/translations/en/common.json:72]: why have you pushed this file, if you changed nothing in it

Comment (Tolik170) [reply src/constants/translations/en/common.json:72]: lint added required space for this file, so, I see no reason to revert changes in this file now

**Review by maarekua — APPROVED** [review]

**Pull request merged**